### PR TITLE
Correction of POS Issue + more efficient n-gram handling + error handling

### PR DESCRIPTION
**Pull request by floriancafiero**

### CODE REVIEW

**Comment by floriancafiero** [comment]
Pom pom pom, it's not anymore :)

File: superstyl/preproc/features_extract.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
+# first line only necessary for older Python versions
 from builtins import sum
 from collections import Counter
 import nltk.tokenize
 import nltk
 import regex as re
 
+#the download is a one time operation: should be placed elsewhere but I put it here for now
+nltk.download('averaged_perceptron_tagger')
```
Comment (Jean-Baptiste-Camps) [inline superstyl/preproc/features_extract.py:9]: Better to put it inside the `if feats == 'pos'`, and then do something like,

````python
try:
    nltk.data.find("mystuff")

except:
    nltk.download("mystuff")
````
cf. https://stackoverflow.com/questions/23704510/how-do-i-test-whether-an-nltk-resource-is-already-installed-on-the-machine-runni

and 

https://stackoverflow.com/questions/30854060/if-a-block-of-code-creates-an-error-do-x-if-not-do-y-python

**Review by Jean-Baptiste-Camps — CHANGES_REQUESTED** [review]
Nice ! I've made a modification request.

In addition, how would you feel about writing a small test for the `pos` option ?

**Comment by Jean-Baptiste-Camps** [comment]
It would have to go there:
https://github.com/SupervisedStylometry/SuperStyl/blob/a98e8871c55edad038ea03288febadffa781fda2/tests/test_load_corpus.py#L221

under the form

````python
# WHEN
corpus, feats = superstyl.load.load_corpus(self.paths, feats="pos", n=2, format="txt")
# THEN
expected_feats = ["something"]
expected_corpus =  ["something"]
self.assertEqual(feats, expected_feats)
self.assertEqual(corpus.to_dict(), expected_corpus)
````

File: superstyl/preproc/features_extract.py (hunk 1 above)
Comment (floriancafiero) [reply superstyl/preproc/features_extract.py:9]: yep, better practice you are right - i was unhappy with my version

**Review by Jean-Baptiste-Camps — CHANGES_REQUESTED** [review]
Would you be ok to write the little test <3 ?

**Pull request merged**